source/index.html.md

```diff
@@ -0,0 +1,429 @@
+---
+title:TPMS API Reference
+
+language_tabs: # must be one of https://git.io/vQNgJ
+  - shell
+ 
+
+toc_footers:
+  - <a href='#'>Sign Up for a Developer Key</a>
+  - <a href='https://github.com/lord/slate'>Documentation Powered by Slate</a>
+
+includes:
+  - errors
+
+search: true
+---
+
+# Introduction
+
+Welcome to the Turntabl Project Management System (TPMS) API. This API documentation contains TPMS API endpoints with regards to Time-entry.
+
+The languages used for TPMS Time entry application are Psql, Java and Angular.
+ 
+
+Some of the technologies used are SAML, Stored Procedure, Aws RDS, etc.
+
+The Application contains various services: Project Service, Employee Service and Logged Hours Service which use same Database.
+
+The Project Service has various functionalities such as add project, list all project, get project by employee ID and delete project
+
+
+
+
+# User 
+```shell
+#FORMAT OF DATA 
+```
+Represents user details.
+
+## User attributes:
+>
+Employee data format:
+>
+
+```json
+
+  {
+  "employee_id": 0,
+  "employee_firstname": "string",
+  "employee_lastname": "string",
+  "employee_email": "string",  
+ } 
+```
+
+
+Name | Type | Description
+---------|---------- | -----------
+employee_id | Number | unique identifier for employee. 
+employee_firstname | String | First Name. 
+employee_lastname | String | Last Name.
+employee_email | String | Email.
+
+
+> 
+Project data format:
```

Comment:
What is a project? A project represents a single unit of work that a person is working on for some time.